libs/island-ui/core/src/lib/Input/Input.mixins.ts

```diff
@@ -55,6 +63,14 @@ export const container = {
 }
 
 export const containerSizes = {
+  xs: {
+    padding: 8,
+    ...themeUtils.responsiveStyle({
+      md: {
+        padding: '10px 16px 10px 10px',
```

Comment:
This seems a bit fishy. Do we not want to use the 8px grid here, i.e.

``padding: `${theme.spacing[1]}px, ${theme.spacing[2]}px, ${theme.spacing[1]}px, ${theme.spacing[1]}px` ``?

Reply:
Designer is taking a look at this one

Reply:
Ditto on this ^

Would also like to see it in the Button component.

Reply:
fixed